src/report.tex

```diff
@@ -818,17 +818,24 @@ \subsubsection{Variance Reduction}
 tally regions.  One consequence of this configuration is that computational
 effort may be expended to increase the neutron population in unimportant regions 
 of neutron phase space.
-
+\\
+\\
 \gls{advantg} only implements reflecting boundaries on the principal axes of a 
```

Comment:
I'll work this sentence into the following paragraph, probably as a PR to your branch.